elections/2023/governance-committee-election.md

```diff
@@ -0,0 +1,107 @@
+
+All exemption forms are private. Only the current governance and election committee members will have access to this information. The governance committee will discard the lists one month after the election.
+
+# Vote
+
+Everyone with voting rights may log into [Helios Voting](https://vote.heliosvoting.org/helios/elections/1ee70ee4-11ce-11ee-aaf8-0a8c9aac83f9/view) using their GitHub account. Voting will be approval voting, where each voter may select up to five candidates. The five candidates with the most votes win the election.
+
+The election committee will accept late registrations to vote and requests to re-send the voting link via email to [cncf-opentelemetry-governance@lists.cncf.io](mailto:cncf-opentelemetry-governance@lists.cncf.io). We encourage pre-registration to minimize the effort required to run this election.
+
+Per Helios, voting is entirely private: nobody will know any individual's vote.
+
+# Results
+
+Voting will close on 25 October 2023 at 00:00 UTC. Nominees will be stack ranked. If a nominee becomes ineligible (for instance, if more than three topmost nominees work for the same company), the election committee will skip those nominees and pick the nominee with the next-highest score. The exact scores for each candidate will be public.
+
+# Schedule
+
+| Date                       | Activity                                                                                                                                   |
+| -------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------ |
+| June 2023                  | This document announced                                                                                                                    |
+| {{TODO DATE}}              | Election blog post and call for nominations                                                                                                |
+| {{TODO DATE}} 23:59 UTC    | End of call for nominations                                                                                                                |
+| {{TODO DATE}}              | Nominees ratified, and Governance Committee ensures nominees' commitment                                                                   |
+| {{TODO DATE}}              | Preliminary list of nominees announced                                                                                                     |
+| {{TODO DATE}}              | List of nominees is finalized on [community](https://github.com/open-telemetry/community) GitHub and advertised via mailing list and Slack |
+| {{TODO DATE}} 23:59 UTC    | Last day to apply for a member of standing exemption                                                                                       |
```

Comment:
I recommend this being a couple of days before the voting period. While it was uneventful to have it the day before in previous elections, I think it's safer to allow for a buffer...